prod-stack.py

```diff
@@ -330,7 +330,7 @@
 # the tasks.
 scheduler_resources = []
 for task in [
-        'Scheduler', 'CertBot', 'StatusDumper', 'DownloadCounter']:
+        'Scheduler', 'SchedulerFullPass', 'CertBot', 'StatusDumper', 'DownloadCounter']:
     scheduler_resources.append(Sub(
```

Comment:
I'll rename this before merging/deploying as it's not really a full pass any more.